packages/mobile/src/components/scrubber/usePosition.ts

```diff
@@ -33,7 +33,7 @@ export const usePosition = (
   }, [])
 
   useEffect(() => {
-    let currentTimeout: number
+    let currentTimeout: NodeJS.Timeout
```

Comment:
lol, @DejayJD  just did the opposite of this a week or two ago

Reply:
yeah TS got angry at me, maybe I have something different locally? 🤔

Reply:
yeah what is going on here? this is from running `npm run verify`... is that what you did on your side @jd?